src/features/canvas/components/monaco-editor.tsx

```
@@ -0,0 +1,122 @@
+interface Props {
+    isOpen: boolean
+    onClose: () => void
+    title: string
+    content: string
+    onSave: (content: string) => void
+}
+
+export default function MonacoEditorModal({ isOpen, onClose, title, content, onSave }: Props) {
+    const [editorContent, setEditorContent] = React.useState(content)
+    const [validation, setValidation] = React.useState({ isValid: true })
+    const editorRef = useRef<any>(null)
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)